### DISCUSSION TITLE
Wireless USB Big Sur Adapter from Unable to Disable SIP

### DISCUSSION THREAD

**Discussion by abdonmuhammad** [chris1111/Wireless-USB-Big-Sur-Adapter#158] [Q&A]
Hi,

I just a got a problem with my internal Wi-Fi MacBook Pro 2015 (A1502). Then, I decide to buy a Wireless Adapter from tp-link with series TL-WN725Nv3. Since my OS is Big Sur, I can not install the regular driver from tp-link. So, I am looking for the driver which is fit for Big Sur. Finally, I found this website. But, I just got stuck when I want to enter the Recovery Mode for turning off the SIP. 

Since I have no connection, I can't enter Recovery Mode. Moreover, I decide to buy LAN Ethernet to USB converter. But, the usb is going to be off when in the enter of Recovery Mode. I have a last chance to get an internet access from Thunderbolt Cable, but it is costly. May I have any recommendation for this problem? Thank you all!

**Comment by chris1111** [comment]
To enter in Recovery you just need a macOS Big Sur USB install drive Create Install Media
 then boot from USB installer

**Reply by chris1111** [reply]
@abdonmuhammad 

free from [appstore](https://apps.apple.com/us/app/macos-big-sur/id1526878132?mt=12)

How to [Create Install Media](https://support.apple.com/en-ca/HT201372)

**Reply by abdonmuhammad** [reply]
Apple silicon
1. Plug the bootable installer into a Mac that is connected to the internet and compatible with the version of macOS you're installing.
2. Turn on your Mac and continue to hold the power button until you see the startup options window, which shows your bootable volumes.
3. Select the volume containing the bootable installer, then click Continue.
4. When the macOS installer opens, follow the onscreen instructions.

This is the instruction from your link. When I can access to terminal for turning off the csrutil?

**Comment by chris1111** [comment]
This the same here but from USB Install Media
[Disable SIP](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/discussions/115)

**Reply by abdonmuhammad** [reply]
Thank you for your response, I will try it. Since, the macOS is large file. I'll update it in the next one hour

**Reply by chris1111** [reply]
> Thank you for your response, I will try it. Since, the macOS is large file. I'll update it in the next one hour


its 12 Gig downloads
your only need **16 Gig USB drive** formate in macOS Extend Journaled / Guid partition Map

**Reply by abdonmuhammad** [reply]
I have a bootable device for installing macOS. What shortcut did I have to press when booting apple logo?

**Comment by abdonmuhammad** [comment]
It solved! Thank you so much. Anyway, why USB Wi-Fi icon show 2 icons in menu bar. Moreover, how to change the icon picture? Thank youu
